scripts/setup-kustomize.sh

```diff
@@ -155,6 +155,9 @@ patches:
     - op: replace
       path: /spec/communal/path
       value: ${COMMUNAL_PATH_PREFIX}${TESTCASE_NAME}
+    - op: add
+      path: /metadata/annotations/vertica.com~1run-nma-in-monolithic-container
+      value: "true"
```

Comment:
I'm thinking maybe we should set this only if deploying with vclusterops.

Reply:
done